### ISSUE TITLE
HW2 Low Accuracy

### ISSUE THREAD

**Issue by chowned** [chowned/Machine_Learning_4IOT#10]
We need to find a strategy to increase to more than 97%

**Comment by Gulisidan** [comment]
pruning_params = {
    'pruning_schedule': tfmot.sparsity.keras.PolynomialDecay(
        initial_sparsity=0.20,
        final_sparsity=final_sparsity,
        begin_step=begin_step,
        end_step=end_step
    )
} 

I am testing these 4 parameter to increase the accuracy

**Comment by Gulisidan** [comment]
I changed 
initial_sparsity = 0.4
final_sparsity = 0.5
alpha=0.14
  'epochs': 60

so I got: 
Accuracy: 97.500%
Model size: 24.7 KB
Preprocessing Latency: 5.1 ms
Model Latency: 0.1 ms
Total Latency: 4.9 ms

**Comment by Gulisidan** [comment]
Closed, final accuracy is 98%, we were able to get this with better HP.
